include/ctr/list.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+typedef struct Item
+{
+	struct Item* next;
+	struct Item* prev;
+} Item;
+
+typedef struct LinkedList
+{
+	Item* first;
+	Item* last;
+	int count;
+} LinkedList;
+
+void LIST_Clear(LinkedList* list);
+void LIST_AddFront(LinkedList* list, Item* item);
+void LIST_AddBack(LinkedList* list, Item* item);
+void* LIST_GetNextItem(Item* item);
+void* LIST_GetFirstItem(LinkedList* list);
+Item* LIST_RemoveMember(LinkedList* L, Item* I);
+Item* LIST_RemoveFront(LinkedList* L);
+Item* LIST_RemoveBack(LinkedList* L);
+void LIST_Init(LinkedList* L, Item* item, int itemSize, int numItems);
```

Comment:
Should we set itemSize as unsigned? Also, try using `const Type*` every time the function doesn't modify the contents that the variable is pointing to

Reply:
I was planning on looking closer at the asm to figure out the signs (I imagine it's fine to make it unsigned, but I wanted to check)